packages/patrol/ios/Classes/PatrolAppServiceClient.swift

```diff
@@ -55,3 +57,36 @@ import NIO
     )
   }
 }
+
+extension Patrol_DartGroupEntry {
+
+  func listTestsFlat(parentGroupName: String) -> [Patrol_DartGroupEntry] {
+    var tests = [Patrol_DartGroupEntry]()
+
+    for test in self.entries {
+      var test = test
```

Comment:
i might be missing something, please explain this to me

Reply:
`test` defined in for-loop body is a `let`, I redeclare it as `var` so I can modify (actually: copy-on-write) its `name` a few lines below